pkg/dockerregistry/server/tagservice_test.go

```diff
@@ -190,6 +191,16 @@ func TestTagCreation(t *testing.T) {
 			continue
 		}
 
+		r = newTestRepository(t, namespace, repo, testRepositoryOptions{
```

Comment:
@dmage suggested this and it fixed the test case :-) still not sure why it was needed.